localization/languages/bn.json

```diff
@@ -131,49 +131,49 @@
         "settingsPreferencesHeading": "পছন্দসমূহ",
         "settingsRestartRequired": "আপনি এই পরিবর্তনগুলি প্রয়োগ করতে পুনরায় শুরু করতে হবে।",
         "settingsPrivacyHeading": "সামগ্রী ব্লকিং",
-        "settingsContentBlockingLevel0": null, //missing translation
-        "settingsContentBlockingLevel1": null, //missing translation
-        "settingsContentBlockingLevel2": null, //missing translation
-        "settingsContentBlockingExceptions": null, //missing translation
+        "settingsContentBlockingLevel0": "সমস্ত বিজ্ঞাপন এবং ট্র্যাকারকে মঞ্জুরি দিন",
+        "settingsContentBlockingLevel1": "তৃতীয় পক্ষের বিজ্ঞাপন এবং ট্র্যাকারদের ব্লক করুন",
+        "settingsContentBlockingLevel2": "সমস্ত বিজ্ঞাপন এবং ট্র্যাকারদের ব্লক করুন",
+        "settingsContentBlockingExceptions": "এই ওয়েবসাইটগুলিতে এখনও বিজ্ঞাপনের অনুমতি দেওয়া হবে:",
         "settingsBlockScriptsToggle": "স্ক্রিপ্ট ব্লক করুন",
         "settingsBlockImagesToggle": "ইমেজ ব্লক করুন",
         "settingsBlockedRequestCount": {
-            "unsafeHTML": null //missing translation
+            "unsafeHTML": "এখনও অবধি, Min <strong>বিজ্ঞাপন</strong> এবং ট্র্যাকারদের অবরুদ্ধ করেছে"
```

Comment:
`<strong></strong>` is filled in with the number of blocked requests, so that it looks like this:
```
So far, Min has blocked <strong>123456</strong> ads and trackers.
```
<img width="409" alt="Screen Shot 2020-10-05 at 10 23 36 AM" src="https://user-images.githubusercontent.com/10314059/95098878-d9280e80-06f4-11eb-98a5-d4eb1205e755.png">

So there shouldn't be any text inside the `strong` tag, as it won't be displayed.